.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,11 @@
 blank_issues_enabled: false
 contact_links:
+  - name: Bug report 🐞
+    url: https://github.com/iTwin/iTwinUI-react/issues/new?labels=bug&template=bug_report.md&projects=iTwin/iTwinUI-react/1
```

Comment:
Have you tried accessing these links with your private account? Because I am wondering whether they work for us as we are owners of the repo.

Reply:
Not sure what you mean.

If you try to create an issue from a template right now, it goes to the same url. I just added the `projects=` part